CleverTapSDK/CleverTap.m

```diff
@@ -1020,6 +1020,10 @@ - (NSDictionary *)batchHeader {
         header[@"regURLs"] = registeredURLSchemes;
     }
     
+    if ([CleverTap getDebugLevel] == 3){
```

Comment:
@AishwaryaNanna please add a comment here, on why this is being done. Refer to [the Android commit here ](https://github.com/CleverTap/clevertap-android-sdk/compare/23a59059916d...fd14730160d8)

Reply:
Added